underscore.js

```diff
@@ -1,3 +1,5 @@
+const { default: remove } = require("./modules/remove");
```

Comment:
This is not what you're supposed to be doing; please add your function to `modules/index.js` instead. You should only edit the sources in `modules/`, not the `underscore.js` or `underscore-esm.js` bundles.

From the fact that you were able to commit this, I can tell that you didn't run `npm install`, because otherwise the commit hook would have prevented it. Please run `npm install` before committing again.